CameraOverlay/orchestrator.py

```diff
@@ -0,0 +1,47 @@
+import json
+import commons
+import paho.mqtt.client as mqtt
+
+brokerIP = "192.168.100.100"
```

Comment:
Would be good to have this as an argument into the program. See https://github.com/Monash-Human-Power/MHP_Modelling/blob/chamaka/pi_code/Python/final%20python/orchestra.py

But happy to do that in a later improvement!

Reply:
good idea, implemented it!